tox.ini

```diff
@@ -95,12 +95,16 @@ setenv =
          #     .decode(getattr(sys.stdout, "encoding", "utf8"))
          # `TypeError: decode() argument 1 must be string, not None`
          PYTHONIOENCODING=utf_8
+
+	 # If no positional arguments are given, try to run the checks on the
+	 # entire codebase, including various pieces of supporting code.
+	 DEFAULT_FILES="src integration static misc setup.py"
 commands =
-         flake8 src integration static misc setup.py
-         python misc/coding_tools/check-umids.py src
-         python misc/coding_tools/check-debugging.py
-         python misc/coding_tools/find-trailing-spaces.py -r src static misc setup.py
-         python misc/coding_tools/check-miscaptures.py
+         flake8 {posargs:{env:DEFAULT_FILES}}
```

Comment:
Arguments to `flake8` are now quoted, and that seems to be wrong:

```
$ flake8 -v 'src integration static misc setup.py'
[...]
flake8.checker            MainProcess    497 INFO     Making checkers
flake8.checker            MainProcess    506 INFO     Checking 0 files
flake8.main.application   MainProcess    507 INFO     Finished running
flake8.main.application   MainProcess    507 INFO     Reporting errors
flake8.main.application   MainProcess    509 INFO     Found a total of 1 violations and reported 0
```

Whereas, previously the arguments where unquoted:

```
$ flake8 -v src integration static misc setup.py
[...]
flake8.checker            MainProcess    462 INFO     Making checkers
flake8.checker            MainProcess   1155 INFO     Checking 342 files
flake8.main.application   MainProcess  23918 INFO     Finished running
flake8.main.application   MainProcess  23919 INFO     Reporting errors
flake8.main.application   MainProcess  24321 INFO     Found a total of 16521 violations and reported 0
```

Also, much of the speedup here seems to be from flake8 not doing what it was supposed to do.

```
$ time flake8 'src integration static misc setup.py'

real	0m0.366s
user	0m0.349s
sys	0m0.016s

$ time flake8 src integration static misc setup.py

real	0m22.703s
user	1m12.801s
sys	0m0.638s
```

Reply:
Ugh.  Thanks for catching that.  I don't know what quoting rules tox uses.  I can't find any documentation. :(  Some experimentation suggests removing the quotes from the `DEFAULT_FILES` definition fixes this, though, so I did that.

The speedup still seems real to me.  For example:

```
[exarkun@baryon:~/Work/python/tahoe-lafs]$ time tox -e codechecks | grep Checking
Checking src...
Checking integration...
Checking static...
Checking misc...
Checking setup.py...

real    0m29.827s
user    1m11.976s
sys     0m1.116s
(_home_exarkun_Work_python_tahoe-lafs) 
[exarkun@baryon:~/Work/python/tahoe-lafs]$ time tox -e codechecks src/allmydata/client.py | grep Checking
Checking src/allmydata/client.py...

real    0m4.664s
user    0m3.962s
sys     0m0.753s
(_home_exarkun_Work_python_tahoe-lafs) 
[exarkun@baryon:~/Work/python/tahoe-lafs]$ 
```